app/models/team_member.rb

```diff
@@ -8,4 +8,28 @@ class TeamMember < ApplicationRecord
   def name
     first_name
   end
+
+  def job_title
+    if discipline == "Development"
+      "Developer"
+    elsif discipline == "Delivery"
+      "Delivery Lead"
+    elsif discipline == "Research"
+      "Researcher"
```

Comment:
```suggestion
      "User Researcher"
```

That's the real job role.